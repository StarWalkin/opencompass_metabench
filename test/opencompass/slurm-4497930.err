  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [1:17:42<00:00, 4662.08s/it]100%|██████████| 1/1 [1:17:42<00:00, 4662.10s/it]
  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:16<00:00, 16.64s/it]100%|██████████| 1/1 [00:16<00:00, 16.64s/it]
