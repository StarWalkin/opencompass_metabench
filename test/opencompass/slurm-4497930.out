/u/zzhang27/.conda/envs/opencompass/bin/python3
08/17 02:42:45 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/17 02:42:45 - OpenCompass - INFO - Loading hf_phi_3_small_8k_instruct: configs/models/phi/hf_phi_3_small_8k_instruct.py
08/17 02:42:45 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/17 02:42:45 - OpenCompass - INFO - Current exp folder: outputs/default/20240817_024245
08/17 02:42:45 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/17 02:42:45 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLInfer[phi-3-small-8k-instruct-hf/gsm8k] on GPU 0
08/17 04:00:27 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLEval[phi-3-small-8k-instruct-hf/gsm8k] on CPU 
dataset    version    metric    mode      phi-3-small-8k-instruct-hf
---------  ---------  --------  ------  ----------------------------
gsm8k      1d7fe4     accuracy  gen                            87.41
08/17 04:00:44 - OpenCompass - INFO - write summary to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240817_024245/summary/summary_20240817_024245.txt
08/17 04:00:44 - OpenCompass - INFO - write csv to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240817_024245/summary/summary_20240817_024245.csv
All processes have been started and their PIDs stored in process_ids_gpt4o.txt.
All processes have completed.
